plugins/coupa-inventory-tracking-management/README.md

```diff
@@ -0,0 +1,13 @@
+---
+description: A plugin that lets certain team members keep track of inventory and stock
+  up accordingly.
+fidelity: IDEA
+name: Inventory Tracking and Management
```

Comment:
This violates our plugin naming convention around verbs

Reply:
Updating the name to `Track your Inventory and Stock`